modules/redpanda/src/test/java/org/testcontainers/redpanda/RedpandaContainerTest.java

```diff
@@ -26,6 +27,7 @@
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.assertj.core.api.Assertions.assertThatThrownBy;
 import static org.assertj.core.api.Assertions.tuple;
+import static org.hamcrest.Matchers.hasItems;
```

Comment:
Should be replaced with Assertj usage.

Reply:
@gustavomonarin are you able to fix this? If not, we can do it. The PR is almost done.

Reply:
Of course. Should be fixed on [7a36f32](https://github.com/testcontainers/testcontainers-java/pull/5994/commits/7a36f32e595d5ef0064b7d4ce780907134c28eaa). 

Let me know if i missed something.

Reply:
The branch is now slightly behind the origin / master. Should i rebase?  (I am not sure if rebase could affect anything with the review process...)

Reply:
no need to do so. I'm waiting for the checks to pass and I will merge it. Thanks @gustavomonarin !